userCode/odwr/dag.py

```diff
@@ -163,24 +168,32 @@ def sta_all_observations(
     async def fetch_obs(datastream: Datastream) -> List[Observation]:
         """Fetch observations for a single datastream and return them."""
         local_observations = []  # the observations array local to this function.
-        range = get_datastream_time_range(datastream.iotid)
+        range: UTCTimeRange = get_datastream_time_range(datastream.iotid)
+
+        assert (
+            range.start.tzinfo == datetime.UTC and range.end.tzinfo == datetime.UTC
+        ), f"{range.start.tzinfo} and {range.end.tzinfo} should be in UTC"
```

Comment:
Will this be a helpful log?

Reply:
Yeah more thinking the print would be something like "PST and UTC should be in UTC"?